template/Dockerfile

```diff
@@ -7,7 +7,6 @@ RUN \
         python3=3.7.3-1 \
         python3-pip=18.1-5 \
         python3-setuptools=40.8.0-1 \
-        python3-pil=5.4.1-2+deb10u2 \
```

Comment:
Should we also remove it from the hassio Dockerfile?

Reply:
Oh, you're right. Thanks!